controlplane/rosa/api/v1beta2/rosacontrolplane_types.go

```diff
@@ -74,6 +74,21 @@ type RosaControlPlaneSpec struct { //nolint: maligned
 	// - ocmApiUrl: Optional, defaults to 'https://api.openshift.com'
 	// +optional
 	CredentialsSecretRef *corev1.LocalObjectReference `json:"credentialsSecretRef,omitempty"`
+
+	AWS *AWSConfiguration `json:"aws,omitempty"`
```

Comment:
Our Spec doesn't really mirror the fields in the ROSA API - I don't see a good reason to diverge from theirs, having to remember the mapping from one to the other is extra mental overhead and as we are, we're putting a bunch of concerns into the same namespace as opposed to having higher-level fields to separate them out.